Geometry/CaloTopology/src/HcalTopology.cc

```diff
@@ -581,8 +576,7 @@ bool HcalTopology::validRaw(const HcalDetId& id) const {
           ok = false;
       }
     } else if (subdet == HcalEndcap) {
-      if ((mode_ == HcalTopologyMode::SLHC) || (mode_ == HcalTopologyMode::H2HE) || (mode_ == HcalTopologyMode::Run3) ||
-          (mode_ == HcalTopologyMode::Run4)) {
+      if (phase2() || (mode_ == HcalTopologyMode::H2HE)) {
```

Comment:
This does not include Run3 now: is it correct? (Just checking)

Reply:
Yes. The logic is with or without HE